src/EnergyPlus/HVACManager.cc

```diff
@@ -2534,225 +2397,222 @@ void ReportAirHeatBalance(EnergyPlusData &state)
+        auto &zoneEquipConfig = state.dataZoneEquip->ZoneEquipConfig(ZoneLoop);
+        auto &refDoorMixing = state.dataHeatBal->RefDoorMixing(ZoneLoop);
         auto &thisZoneHB = state.dataZoneTempPredictorCorrector->zoneHeatBalance(ZoneLoop);
 
         // Break the infiltration load into heat gain and loss components
         ADSCorrectionFactor = 1.0;
 
         if (state.afn->simulation_control.type == AirflowNetwork::ControlType::MultizoneWithDistributionOnlyDuringFanOperation) {
             // CR7608 IF (TurnFansOn .AND. AirflowNetworkZoneFlag(ZoneLoop)) ADSCorrectionFactor=0
-            if ((state.dataZoneEquip->ZoneEquipAvail(ZoneLoop) == CycleOn || state.dataZoneEquip->ZoneEquipAvail(ZoneLoop) == CycleOnZoneFansOnly) &&
+            if ((state.dataZoneEquip->ZoneEquipAvail(ZoneLoop) == DataHVACGlobals::CycleOn ||
+                 state.dataZoneEquip->ZoneEquipAvail(ZoneLoop) == DataHVACGlobals::CycleOnZoneFansOnly) &&
                 state.afn->AirflowNetworkZoneFlag(ZoneLoop))
                 ADSCorrectionFactor = 0.0;
         }
 
-        if (thisZoneHB.MAT > Zone(ZoneLoop).OutDryBulbTemp) {
+        if (thisZoneHB.MAT > zone.OutDryBulbTemp) {
 
-            ZnAirRpt(ZoneLoop).InfilHeatLoss =
-                thisZoneHB.MCPI * (thisZoneHB.MAT - Zone(ZoneLoop).OutDryBulbTemp) * TimeStepSysSec * ADSCorrectionFactor;
-            ZnAirRpt(ZoneLoop).InfilHeatGain = 0.0;
+            znAirRpt.InfilHeatLoss = thisZoneHB.MCPI * (thisZoneHB.MAT - zone.OutDryBulbTemp) * TimeStepSysSec * ADSCorrectionFactor;
+            znAirRpt.InfilHeatGain = 0.0;
 
-        } else if (thisZoneHB.MAT <= Zone(ZoneLoop).OutDryBulbTemp) {
+        } else {
 
-            ZnAirRpt(ZoneLoop).InfilHeatGain =
-                thisZoneHB.MCPI * (Zone(ZoneLoop).OutDryBulbTemp - thisZoneHB.MAT) * TimeStepSysSec * ADSCorrectionFactor;
-            ZnAirRpt(ZoneLoop).InfilHeatLoss = 0.0;
+            znAirRpt.InfilHeatGain = thisZoneHB.MCPI * (zone.OutDryBulbTemp - thisZoneHB.MAT) * TimeStepSysSec * ADSCorrectionFactor;
+            znAirRpt.InfilHeatLoss = 0.0;
         }
         // Report infiltration latent gains and losses
-        CpAir = PsyCpAirFnW(state.dataEnvrn->OutHumRat);
-        H2OHtOfVap = PsyHgAirFnWTdb(thisZoneHB.ZoneAirHumRat, thisZoneHB.MAT);
+        CpAir = Psychrometrics::PsyCpAirFnW(state.dataEnvrn->OutHumRat);
+        H2OHtOfVap = Psychrometrics::PsyHgAirFnWTdb(thisZoneHB.ZoneAirHumRat, thisZoneHB.MAT);
         if (thisZoneHB.ZoneAirHumRat > state.dataEnvrn->OutHumRat) {
 
-            ZnAirRpt(ZoneLoop).InfilLatentLoss =
+            znAirRpt.InfilLatentLoss =
                 thisZoneHB.MCPI / CpAir * (thisZoneHB.ZoneAirHumRat - state.dataEnvrn->OutHumRat) * H2OHtOfVap * TimeStepSysSec * ADSCorrectionFactor;
-            ZnAirRpt(ZoneLoop).InfilLatentGain = 0.0;
+            znAirRpt.InfilLatentGain = 0.0;
 
-        } else if (thisZoneHB.ZoneAirHumRat <= state.dataEnvrn->OutHumRat) {
+        } else {
```

Comment:
[src/EnergyPlus/HVACManager.cc:2570]:(style),[multiCondition],Expression is always true because 'else if' condition is opposite to previous condition at line 2564.

Reply:
:+1: